python/examples/handler.py

```diff
@@ -56,20 +47,15 @@ def echo(request, response, proxy):
     print "echo"
     yield tornado.gen.sleep(1)
     # stream args right back to request side
-    response.argstreams = [
-        InMemStream(request.endpoint),
-        request.argstreams[1],
-        request.argstreams[2]
-    ]
+    response.set_header(request.get_header_s())
```

Comment:
I think this should probably be:

`response.set_headers()` - takes a list of headers to set

and

`response.add_header()` - add a single header